extensions/Worker.Extensions.ServiceBus/src/ServiceBusMessageActions.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus;
+using Google.Protobuf.WellKnownTypes;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.ServiceBus.Grpc;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    [InputConverter(typeof(ServiceBusMessageActionsConverter))]
+    public class ServiceBusMessageActions
+    {
+        private readonly Settlement.SettlementClient _settlement;
+
+        internal ServiceBusMessageActions(Settlement.SettlementClient settlement)
+        {
+            _settlement = settlement;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServiceBusMessageActions"/> class for mocking use in testing.
+        /// </summary>
+        /// <remarks>
+        /// This constructor exists only to support mocking. When used, class state is not fully initialized, and
+        /// will not function correctly; virtual members are meant to be mocked.
+        ///</remarks>
+        protected ServiceBusMessageActions()
```

Comment:
This is not currently being leveraged. Considered removing?

Reply:
This is to support mocking.

Reply:
Sounds good. Unless I missed it, I don't think tests were in yesterday when asked.

Reply:
Yeah it is actually a pattern we follow so that customers can mock the client types.